docs/source/deployment-considerations.rst

```diff
@@ -0,0 +1,91 @@
+Deployment Considerations
+=========================
+
+Thanks to the efforts of the open-source community, there are tools to deploy Dask :ref:`pretty much anywhere <deployment-options>`—if you can get computers to talk to each other, you can probably turn them into a Dask cluster.
+
+**However, getting Dask running is often not the last step, but the first step.** This document attempts to cover some of the things *outside of Dask* you may have to think about when managing a Dask deployment.
```

Comment:
I like how this is clearly setting the stage that these things are out of scope or on the periphery for Dask.